docker-compose.yml

```diff
@@ -1,37 +1,47 @@
-version: '3.3'
+version: '3.8'
 
 services:
 
   database:
     image: mysql
+    container_name: exip_db
     restart: always
     command: --default-authentication-plugin=mysql_native_password
     ports:
-      - "3306:3306"
+      - "${DATABASE_PORT}:${DATABASE_PORT}"
     volumes:
-      - ./database/exip.sql:/docker-entrypoint-initdb.d/database.sql
+      - ./database/exip.sql:/docker-entrypoint-nitdb.d/database.sql:rw
```

Comment:
is it not called `initdb` anymore?

Reply:
A good spot, must have been an accidental delete.